docs/faq/nuclei.md

```diff
@@ -72,3 +72,24 @@
 ??? info "I have more questions! 🙋"
 	
 	Please join our [Discord server](https://discord.gg/projectdiscovery), or contact us via [Twitter](http://twitter.com/pdnuclei).
+
+
+??? warning "Problem related to dependencies for using nuclei on Ubuntu AMD64 and Docker"
+
+	If you are using nuclei on a machine running Ubuntu AMD64 or in a Docker environment, you may encounter issues with headless browsers. To avoid these issues, you can follow these steps to install the necessary dependencies:
```

Comment:
We need to describe which issues, for example, that the browser might not be able to run due to missing dependencies related to specific OS-shared libraries. Pre-installing the browser for the distribution might fix these missing dependencies errors.
Additionally, it is helpful to report an example of typical error messages that help search engines correctly index the content.